stop_and_wait_plugin/config/parameters.yaml

```diff
@@ -16,7 +16,7 @@ accel_limit_multiplier : 0.95
 
 # Double: Minimum speed the vehicle can command before being ready to stop
 # Units: m/s
-crawl_speed : 2.2352
+crawl_speed : 0.89
```

Comment:
See comment
https://github.com/usdot-fhwa-stol/carma-platform/pull/1573#discussion_r777481503

Reply:
This number should be 0.91 per the discussion on the linked PR. Did that PR not get merged into this branch?